documentation/docs/decentralized_identifiers/update.mdx

```diff
@@ -12,19 +12,382 @@ keywords:
 ---
 import account_manipulate_did_rs from '!!raw-loader!../../../examples/account/manipulate_did.rs';
 import CodeSnippet from '../../src/components/CodeSnippetComponent'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
 
 
-TODO: Explain how DID Documents can be manipulated and how updates should be published.
-
 ## Example
 
 This example shows how you can add more to an existing DID Document. The two main things you can add are Verification Methods and Services. A verification method adds public keys, which can be used to digitally sign things like an identity. The services provide metadata around the identity via URIs. These can be URLs, but also emails or IOTA indices.
 
-### Account Module (Recommended)
-
 <CodeSnippet 
     nodeReplitLink="https://repl.it/@IOTAFoundation/manipulate-did?lite=true"
     rustContent={account_manipulate_did_rs}
     nodeGithubLink = "https://github.com/iotaledger/identity.rs/blob/dev/bindings/wasm/examples-account/src/manipulate_did.ts"
     rustGithubLink = "https://github.com/iotaledger/identity.rs/blob/dev/examples/account/manipulate_did.rs"
 />
+
+### 1. Creating Identity
+
+The Example above starts by creating an identity using the account.
+See [create identity example](./create.mdx).
+
+<Tabs>
+<TabItem value="Rust" label="Rust">
+
+```rust
```

Comment:
Can/should we link the state of the tabs across the page, such that when I select Node.js, it will show Node.js code on all of them?

Reply:
It's possible, it needs a React component for that, If we decide to keep the small snippets (since they might be hard to maintain), I can improve them by creating an easy-to-use component in a separate PR.

Reply:
I am happy to keep them as this is a pretty crucial article and a bit of extra effort to make it really clear for the devs is worth it.